hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/delegation/SQLDelegationTokenSecretManager.java

```diff
@@ -458,8 +458,8 @@ public int incrementCurrentKeyId() {
   // Token operations in SQL database
   protected abstract byte[] selectTokenInfo(int sequenceNum, byte[] tokenIdentifier)
       throws SQLException;
-  protected abstract Map<byte[], byte[]> selectTokenInfos(long maxModifiedTime, int topResults)
-      throws SQLException;
+  protected abstract Map<byte[], byte[]> selectStaleTokenInfos(long maxModifiedTime,
+      int topResults) throws SQLException;
```

Comment:
Nit: `topResults` implies ordering. Maybe use `maxResults` like you did for the implementation.

Reply:
Makes sense, I've renamed it